clipper_admin/clipper_admin/sample_config.yaml

```diff
@@ -0,0 +1,46 @@
+Model1:
+  obj_type: "model"
+  input_type: "integers"
+  version: "1.2.7"
+  image: "test-model:1"
+
+Application1:
+  obj_type: "application"
+  input_type: "doubles"
+  slo: 10000
+  default_value: "not received"
+
+Model2:
+  obj_type: "model"
+  input_type: "floats"
+  version: "1.2.7"
+  image: "train-model:2"
+  labels: ["second model configured", "working successfully"]
+  num_replicas: 2
```

Comment:
Models should support the `batch_size` field as well. See https://github.com/ucbrise/clipper/blob/137cc852a5c9ed5cca11c6c2f89f1e708a561198/clipper_admin/clipper_admin/clipper_admin.py#L403